### PR TITLE
Write ember-cli-page-object migration post

### PR DESCRIPTION
**Pull request by juanazam**

### CODE REVIEW

File: source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+  page.modal().title() //  transaltes to `find('#my-modal h3').text()`
+{% endcodeblock %}
+
+On `v1` we decided to implement scope inheritance, this means that if a
+component defines a scope and has a child component, the latter will
+inherit its parent scope.
+
+{% codeblock lang:js %}
+  page.title; // translates into find('#my-page h1').text()
+  page.modal.title //  transaltes into find('#my-page #my-modalh3').text()
```
Comment (jmbejar) [inline source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown]: Missing whitespace in `#my-modalh3` (should be `#my-modal h3`)

Comment (jmbejar) [inline source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown]: Also, just noticed you're sometimes using the ` symbol after "translates to" but here it's not being used

File: source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+<div class="#my-page">
+  <h1>My title</h1>
+  <h1>My other title</h1>
+</div>
+{% endcodeblock %}
+
+The following call will throw an error:
+
+{% codeblock lang:js %}
+  page.title; // Kaboom!
```
Comment (jmbejar) [inline source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown]: I would put here the error message as it appears in the console.

`Error: Assertion Failed: "#my-page h1" matched more than one element. If this is not an error use { multiple: true }`

**Comment by jmbejar** [comment]
Here goes my general :+1: to this post :fireworks: 

Probably, the missing whitespace is the only mandatory issue to fix before going live.

File: source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+
+As you may know by now, `ember-cli-page-object`'s latest version was
+released, moving to a more polished and mature API and also defining
+more intuitive behavior to every aspect of its functionality.
+
+<!--more-->
+
+This sounds great, but it also comes with a cost, the cost of migrating
+your tests suite. You can check the list of changes `v1` included over
+[here](https://github.com/san650/ember-cli-page-object/pull/91), on this
```
Comment (jmbejar) [inline source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown]: I would start a new sentence here: 

> You can check the list of changes v1 included over **here. On** this post I will go...

**Comment by san650** [comment]
@juanazam @jmbejar I'm not sure if this post makes any sense now, because we already published most of it in the website: http://ember-cli-page-object.js.org/docs/v1.1.x/migrating

@juanazam should we close it?

**Pull request closed without merging**